### PR TITLE
[ORCA-84] Added content to the Deployment Information page.

### PR DESCRIPTION
**Pull request by Quarriers89**
## Summary of Changes

Created content for the Deployment Information page. Included descriptions and links for deploying ORCA and its prerequisites.

Addresses [ORCA-84: Create "How to Deploy ORCA" page for ORCA developer documentation.](https://bugs.earthdata.nasa.gov/browse/ORCA-84)

## Changes

* Updated title to "Deployment Information"
* Updated description of the page to "Provides developer information for ORCA code deployment."
* Added content for the Deploying ORCA section, including subsections critical to ORCA deployment.
* Added content for the Configuring ORCA section.

## Type of change

Please delete options that are not relevant.

- [X] This change requires a documentation update

## How Has This Been Tested?

N/A

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [X] User documentation
 - [X] My changes generate no new warnings

### CODE REVIEW

File: website/docs/developer/deployment-guide/deployment.md [hunk 1]
```diff
@@ -1,8 +1,51 @@
...
+
+### Deploy ORCA with Cumulus
+After the ORCA archive bucket has been created, ORCA is ready to be
+deployed. Visit [ORCA deployment documentation](deployment-with-cumulus.md)
+for instructions to deploy ORCA with Cumulus.
+
+
+## Configuring ORCA
+Once ORCA is deployed, configure the collection in the Cumulus 
+Dashboard for utilizing ORCA. See ORCA Configuration for
```
Comment (bhazuka) [inline website/docs/developer/deployment-guide/deployment.md:49]: ORCA Configuration can be a link like the following.

`[ORCA Configuration](deployment-with-cumulus.md#collection-configuration)`

I think.

**Review by bhazuka — CHANGES_REQUESTED** [review]
Looks good, minor comment for adding a link if you wish.

File: website/docs/developer/deployment-guide/deployment.md [hunk 2]
```diff
@@ -1,8 +1,51 @@
 ---
 id: deployment
-title: Developer Deployment
+title: Deployment Information
 description: Provides developer information for ORCA code deployment.
 ---
 
-This document is currently under construction.
+## Introduction to ORCA
```
Comment (AndrewDorn) [inline website/docs/developer/deployment-guide/deployment.md:7]: Looks like this was copied from orca-intro. Don't think it's needed here.

File: website/docs/developer/deployment-guide/deployment.md [hunk 3]
```diff
@@ -1,8 +1,51 @@
...
+
+Visit [Creating a Deployment Environment](setting-up-deployment-environment.md)
+to view the instructions for setting up your ORCA deployment environment.
+
+### ORCA Archive Bucket
+ORCA maintains a cloud ready backup of science and 
+ancillary data in an S3-IA bucket and utilizes AWS 
+bucket transition policies to store the data in S3 
+Glacier for the long term. The ORCA archive bucket can 
+live in any NGAP Organizational Unit (OU).
```
Comment (AndrewDorn) [inline website/docs/developer/deployment-guide/deployment.md:30]: Should we add a couple of these terms to the glossary.md?

**Review by AndrewDorn — CHANGES_REQUESTED** [review]
Seems like there's some extra introductory information that can be removed.

File: website/docs/developer/deployment-guide/deployment.md (hunk 2 above)
Comment (Quarriers89) [reply website/docs/developer/deployment-guide/deployment.md:7]: Removed this section.

File: website/docs/developer/deployment-guide/deployment.md (hunk 3 above)
Comment (Quarriers89) [reply website/docs/developer/deployment-guide/deployment.md:30]: will put a new card into the backlog for OU and NGAP

**Review by bhazuka — APPROVED** [review]

**Pull request merged**